src/libcollections/borrow.rs

```diff
@@ -185,6 +229,18 @@ impl<'a, B: ?Sized> Cow<'a, B> where B: ToOwned {
     /// Extract the owned data.
     ///
     /// Copies the data if it is not already owned.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::borrow::Cow;
+    ///
+    /// let cow: Cow<&str> = Cow::Owned("Hello");
```

Comment:
My comment above about how `&str` is not super common for the `Owned` type goes for here as well, perhaps this could be `Cow<str>` and use `Cow::Borrowed` instead?